crates/driver/src/domain/liquidity/zeroex.rs

```diff
@@ -10,6 +10,7 @@ use {
 #[derive(Clone, Debug)]
 pub struct LimitOrder {
     pub order: Order,
+    pub fillable_amounts: FillableAmounts,
```

Comment:
nanonit: remaining/fillable for conciseness (it's clear from the type that it's amounts)?